packages/spaces/src/Spaces.js

```diff
@@ -378,6 +385,7 @@ Spaces.defaultProps = {
   spaceIds: [],
   payerIds: [],
   spaces: [],
+  operationName: 'PuiBootstrapSpacesAnonymousOperation',
```

Comment:
This needs to match the query operation name. By default, that is configurationFindMany. That's the why the logic around operationName in element got more complicated: https://github.com/Availity/element/commit/8189c94dd6b8fef981266d121b3a2893639fa482#diff-67f7809a696b9da44542461492e329085e020f353ca71a466759e269bf9cc8b1